client/chat/action-creators.ts

```diff
@@ -110,31 +100,31 @@ export function retrieveUserList(channel) {
 
     const params = { channel }
     dispatch({
-      type: CHAT_LOAD_USER_LIST_BEGIN,
+      type: '@chat/retrieveUserListBegin',
       payload: params,
     })
     dispatch({
-      type: CHAT_LOAD_USER_LIST,
-      payload: fetch(apiUrl`chat/${channel}/users`, { method: 'GET' }),
+      type: '@chat/retrieveUserList',
+      payload: fetch<ChatUser[]>(apiUrl`chat/${channel}/users`, { method: 'GET' }),
       meta: params,
     })
   }
 }
 
-export function activateChannel(channel) {
+export function activateChannel(channel: string) {
```

Comment:
These need return types so they actually get typechecked against the action structure